src/View/SSViewer.php

```diff
@@ -552,11 +599,13 @@ public function includeRequirements($incl = true)
      * @param ViewableData $item The item to use as the root scope for the template
      * @param array $overlay Any variables to layer on top of the scope
      * @param array $underlay Any variables to layer underneath the scope
-     * @param ViewableData $inheritedScope The current scope of a parent template including a sub-template
+     * @param SSViewer_Scope|null $inheritedScope The current scope of a parent template including a sub-template
```

Comment:
Just a PHPDoc so doesn't break anything to change this - and I'm changing it to what is actually being passed in here.